spec/factories/invoice_items.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do 
+  factory :invoice_item, class: InvoiceItem do 
+    association :item
+    association :invoice
```

Comment:
I've done this by simply passing `invoice` without the `association`, I'll have to look into what the association does